### PR TITLE
Favicon, Footer & CNCF logo

### PR DESCRIPTION
**Pull request by schristoff**

### CODE REVIEW

File: docs/layouts/index.html [hunk 1]
```diff
@@ -92,6 +90,16 @@ <h2 class="mt-6 text-xl mx-auto text-center font-bold text-gray-400 leading-6 tr
     </section>
   </div>
 </div>
+<div>
+  <div class="mt-6 grid grid-cols-1 gap-4 w-full">
+    <h2 class="mt-6 text-xl mx-auto text-center font-bold text-gray-400 leading-6 tracking-tighter bg-clip-text text-transparent bg-gradient-to-b from-gray-900 to-gray-600 dark:from-gray-100 dark:to-gray-400" style="max-width: 800px">
+     We are a <a href="https://www.cncf.io/">Cloud Native Computing Foundation</a> Sandbox Project
+     <br>
```
Comment (schristoff) [inline docs/layouts/index.html:97]: I have no idea how to do this without a bunch of `<br>`s to make spacing nice.

File: docs/layouts/index.html [hunk 2]
```diff
@@ -92,6 +90,16 @@ <h2 class="mt-6 text-xl mx-auto text-center font-bold text-gray-400 leading-6 tr
...
+     <br>
+     <center> <img href="https://www.cncf.io/" src="/images/cncf-color.svg" alt="We are a Cloud Native Computing Foundation Sandbox Project" width="350" height="350"> </center>
```
Comment (schristoff) [inline docs/layouts/index.html:99]: Another problem is in dark mode this gets washed out, there is a black and white CNCF logo but I don't know how to switch it over when in dark mode

File: docs/layouts/index.html (hunk 1 above)
Comment (zelfroster) [reply docs/layouts/index.html:97]: I might have also done the same. If the footer looks good to you, It's good then. Also, should we also give credit to Hextra by adding the `Powered by Hextra` text in footer?

File: docs/layouts/index.html (hunk 2 above)
Comment (zelfroster) [reply docs/layouts/index.html:99]: I will fix this.

File: docs/layouts/index.html (hunk 1 above)
Comment (schristoff) [reply docs/layouts/index.html:97]: Yeah, we should give credit to Hextra, you're right. I'll turn back on the "powered by" in the config in a second :)

**Comment by zelfroster** [comment]
Should I merge this now? I can do any further changes on my end.

**Comment by schristoff** [comment]
good to merge now!

**Pull request merged**